Install_Instructions.md

```diff
@@ -0,0 +1,43 @@
+# Tux Typing 1.8.3 Installation Guide
+
+**Release Date:** 19 Aug 2014  
+**Website:** [tux4kids.com](http://www.tux4kids.com)  
+**Developer Contact:** tux4kids-tuxtype-dev@lists.alioth.debian.org
+
+## Overview
+Tux Typing is an educational typing tutor for children starring Tux, the Linux penguin. This guide provides installation instructions for Windows, macOS, and Ubuntu.
+
+## Windows Installation
+
+1. **Download Installer:**
+   - Download the Tux Typing installer for Windows from the [official website](http://www.tux4kids.com).
+
+2. **Run Installer:**
+   - Double-click the downloaded `.exe` file to run the installer.
+   - Follow the on-screen instructions to complete the installation.
+
+3. **Launch Tux Typing:**
+   - After installation, you can launch Tux Typing from the Start Menu or the desktop shortcut.
+
+## macOS Installation
+
+1. **Download Installer:**
+   - Download the Tux Typing installer for macOS from the [official website](http://www.tux4kids.com).
+
+2. **Install Application:**
+   - Open the downloaded `.dmg` file.
+   - Drag the Tux Typing application to the Applications folder.
+
+3. **Launch Tux Typing:**
+   - Open the Applications folder and double-click Tux Typing to launch it.
+
+## Ubuntu Installation
+
+### Option 1: Using the Package Manager
```

Comment:
If there is not an option 2, then I don't think this section header needs to be here 😉